cypress/compoment/NavBar.cy.jsx

```diff
@@ -0,0 +1,76 @@
+import NavBar from "../../src/components/NavBar/NavBar";
+import { BrowserRouter as Router } from "react-router-dom";
+
+describe("Render <NavBar /> component", () => {
```

Comment:
I don't think this test suite is very helpful.

1. There is so much code duplication. Imagine that the color changes for some reason, you would have to rewrite the tests. Not maintainable.
2. The colors don't even correspond to the actual code. How am I supposed to know what `rgb(163, 163, 163)` is at first glance?
3. You added extra classes with confusing logic. It took me a while to understand what you were doing.

Something like this instead...
```
  it("icon is inactive by default", () => {
      cy.get('icon').should('have.class', 'icon-inactive');
  });

it("icon is active when clicked", () => {
      cy.get('icon')
      .click()
      .should('have.class', 'icon-active');
  });
```